ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -239,6 +239,107 @@ device_classes: !ZenPackSpec
                                 rrdmin: 0
                             missing:
                                 rrdmin: 0
+                graphs:
+                    Utilization:
+                        units: percent
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            size:
+                                dpName: volume_size
+                                lineType: DONTDRAW
+                            free:
```

Comment:
It seems that the `size,-,size,/,-100,*` is transforming the volume_free datapoint into "percent used". So the label for this graph point should be updated to "Used". Also be sure to follow our convention of capitalizing graph point labels.